client/src/components/AssignPersonModal.js

```diff
@@ -94,9 +98,17 @@ const AssignPersonModal = ({ position, showModal, onCancel, onSuccess }) => {
         </>
       )
       newError = { message: errorMessage }
+    } else if (!isAdvisor() && (removeUser || !person)) {
+      const errorMessage = (
+        <>
+          If you save, type of the <b>{position.name}</b> position is going to
+          be converted to <b>{Position.TYPE.ADVISOR}</b>.
```

Comment:
`Position.TYPE.ADVISOR` is a code-reference, not very meaningful to the actual user. Perhaps better described as `a regular (i.e. non-privileged) advisor position`.